lib/stdlib/src/gen_event.erl

```diff
@@ -1,7 +1,7 @@
 %%
 %% %CopyrightBegin%
 %%
-%% Copyright Ericsson AB 1996-2014. All Rights Reserved.
+%% Copyright Ericsson AB 1996-2015. All Rights Reserved.
```

Comment:
Why is the copyright changed to 2015 here and 2016 in lib/stdlib/doc/src/Makefile?